src/ontology/mondo-edit.obo

```diff
@@ -277866,6 +277900,7 @@ def: "Autosomal recessive limb-girdle muscular dystrophy type 2T (LGMD2T) is a f
 subset: ordo_disease {source="Orphanet:363623"}
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in GMPPB" EXACT [MONDO:design_pattern]
 synonym: "GMPPB autosomal recessive limb-girdle muscular dystrophy" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "LGMD-GMPPB related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/